docs/console/guides/local-network/subnet.md

```diff
@@ -0,0 +1,173 @@
++===============+=============+=================+===============+=======+==================+=========+===================+
+| my-subnet-evm | 542c...f94e | AvalancheSubnet | aws/us-east-1 | Small | 2024-01-30T13:58 | Pending |  ID         |     |
+|               |             |                 |               |       |                  |         |  Validators | 0   |
++---------------+-------------+-----------------+---------------+-------+------------------+---------+-------------------+
+```
+
+:::info
+Unlike the `avalancheNode` resource, there is no concept of `size` for the `avalancheSubnet` resource. The Ash Console will create a small instance (e.g.: `t2.micro`) that will serve as the Subnet control plane.
+:::
+
+## Subnet ID
+
+After a few minutes, we can get the Subnet ID of our Subnet with the `console resource info` command:
+
+```bash title="Command"
+ash console resource info my-subnet-evm
+```
+
+```bash title="Output"
+Resource 'my-subnet-evm' of project 'ash-devnet':
++---------------+-------------+-----------------+---------------+-------+------------------+---------+------------------------------------------------------------------+
+| Resource name | Resource ID | Type            | Cloud region  | Size  | Created at       | Status  | Resource specific                                                |
++===============+=============+=================+===============+=======+==================+=========+==================================================================+
+| my-subnet-evm | 525c...c33e | AvalancheSubnet | aws/us-east-1 | Small | 2024-01-30T16:05 | Running |  ID         | VDyFNft1FnKQn1bxZfLsjshokhQRP1j4cTBjhob89CFFLcrRE  |
+|               |             |                 |               |       |                  |         |  Validators | 5                                                  |
++---------------+-------------+-----------------+---------------+-------+------------------+---------+------------------------------------------------------------------+
+```
+
+## Blockchain ID
+
+From here, we can query one of our nodes to get information about the Subnet:
+
+```bash title="Command"
+ash avax subnet info VDyFNft1FnKQn1bxZfLsjshokhQRP1j4cTBjhob89CFFLcrRE
+```
+
+```bash title="Output"
+Subnet 'VDyFNft1FnKQn1bxZfLsjshokhQRP1j4cTBjhob89CFFLcrRE':
+  Type: Permissioned
+  Control keys: ["P-local18jma8ppw3nhx5r4ap8clazz0dps7rv5u00z96u"]
+  Threshold:    1
+  Blockchains list (1): 
+  - 'AshLocalEVM':
+    ID:      2VgLztfZWQewKVqtsBVpdbMdEFWScDgcwP21UWQwifomCG8weF
+    VM ID:   srEXiWaHuhNyGwPUi444Tu47ZEDwxTWrbQiuD7FmgSAQ6X7Dy
+    VM type: SubnetEVM
+  Validators list (5): 
+  - NodeID-7Xhw2mDxuDS44j42TCB6U5579esbSt3Lg
+  - NodeID-GWPcbFJZFfZreETSoWjPimr846mXEKCtu
+  - NodeID-NFBbbJ4qCmNaCzeW7sxErhvWqvEQMnYcN
+  - NodeID-MFrZFVCXPv5iCn6M9K6XduxGTYp891xXZ
+  - NodeID-P7oB2McjBGgW2NXXWVYjV8JEDFoW9xDE5
+```
```

Comment:
Same comments as above

Reply:
Done in https://github.com/AshAvalanche/ash-docs/pull/79/commits/6df71bc71fe5f55f263dfd532ef4b6b885d72d2b